lib/amqp/Vhost.js

```diff
@@ -322,7 +323,29 @@ function Vhost(config) {
         return channel.close();
       }
 
-      once(null, channel);
+      channel.consume(
+        'amq.rabbitmq.reply-to',
+        function(message) {
+          const { correlationId } = message.properties;
+
+          const replyHandler = self.replyHandlers[correlationId];
+
+          if (!replyHandler) return;
+
+          let content = message.content.toString();
+
+          try {
+            content = JSON.parse(content);
```

Comment:
I think there's probably a better way to define `content`

Reply:
We should use the same approach as for normal subscriptions. https://github.com/guidesmiths/rascal/blob/master/lib/amqp/Subscription.js#L91-L94